index.js

```diff
@@ -25,6 +25,31 @@ app.get('/api/v1/favorites', (request, response) => {
     });
 });
 
+app.post('/api/v1/favorites', (request, response) => {
+  const favorite = request.body;
+  for (let requiredParameter of ['name', 'artist_name', 'genre', 'rating']) {
```

Comment:
Sad path testing looks good. I think the front end will be taking care of the 1-100 restriction on the rating, which we talked about!